core/src/CMakeLists.txt

```
@@ -8,7 +8,9 @@ KOKKOS_INCLUDE_DIRECTORIES(
 INSTALL (DIRECTORY
   "${CMAKE_CURRENT_SOURCE_DIR}/"
   DESTINATION ${KOKKOS_HEADER_DIR}
-  FILES_MATCHING PATTERN "*.hpp"
+  FILES_MATCHING 
```

```suggestion
  FILES_MATCHING
```
CI caught this trailing whitespace